src/algorithms/tracking/OrthogonalTrackSeedingConfig.h

```diff
@@ -78,5 +78,13 @@ namespace eicrecon {
     float  m_seedConfMaxZOrigin_forw = 150.0 * Acts::UnitConstants::mm;
     float  m_minImpactSeedConf_forw = 1.0 * Acts::UnitConstants::mm;
 
+    //////////////////////////////////////
+    ///Seed Covariance Error Matrix
+    float m_loc_a_Error = 1.5;     //Error on Loc a
+    float m_loc_b_Error = 1.5;     //Error on Loc b
+    float m_phi_Error = 0.02;      //Error on phi
+    float m_theta_Error = 0.002;   //Error on theta
+    float m_qOverP_Error = 0.025;  //Error on q over p
+    float m_time_Error = 0.1;      //Error on time
```

Comment:
Physical quantities should have units.
```suggestion
    float m_loc_a_Error = 1.5 * Acts::UnitConstants::mm;     //Error on Loc a
    float m_loc_b_Error = 1.5 * Acts::UnitConstants::mm;     //Error on Loc b
    float m_phi_Error = 0.02 * Acts::UnitConstants::rad;     //Error on phi
    float m_theta_Error = 0.002 * Acts::UnitConstants::rad;  //Error on theta
    float m_qOverP_Error = 0.025 / Acts::UnitConstants::GeV; //Error on q over p
    float m_time_Error = 0.1 * Acts::UnitConstants::mm;      //Error on time
```
These units are such that they are multiplying by 1 (Acts uses units of mm, rad, GeV, and time is defined in terms of distance, https://acts.readthedocs.io/en/latest/core/definitions/units.html).

We may want to consider (in the future) changing the last line to:
```
    float m_time_Error = 0.333 * Acts::UnitConstants::ps;      //Error on time
```
This is what you have there now (but someone should double-check my algebra).

Reply:
I understand wanting to put units here. But aren't we in principle 'double applying' the units since they are used here as well:

https://github.com/eic/EICrecon/blob/main/src/algorithms/tracking/CKFTracking.cc#L70-L75

Since `Acts::UnitConstants::mm`, 'Acts::UnitConstants::rad', and 'Acts::UnitConstants::GeV' are all equal to 1, this won't have any practical effect.

This may be important to consider for the time in the future, since we currently mulitply by 'Acts::UnitConstants::ns', in CKFTracking which is not equal to 1. Right now, we have

`float m_time_Error = 0.1`

which I guess is getting converted in the CKF tracking to `0.1ns = 0.1 * 1e-9 * 3e11 = 30mm`.

Reply:
@wdconinc. What do you think about my comment yesterday? If you think it is best to add the mm, rad, GeV units to the values in the configuration file, I'll do that. Thanks.

Reply:
We should apply units where the numerical dimensioned quantities are specified. The units in `edm4eic_indexed_units` are not units but unit **conversion** ratio when going from EDM4eic quantities to Acts quantities. The rules are:
- when you write a number, you need to multiply with a unit,
- when you change from one unit domain (EDM4eic, Acts, DD4hep) to another, you need a unit conversion ratio.

In the following line:
https://github.com/eic/EICrecon/blob/ac3821d47eb4cbd6f39bddaac19710a7840104e0/src/algorithms/tracking/CKFTracking.cc#L169
we change from EDM4eic to Acts unit domains with the unit conversion ratios.

In e.g. the following line:
https://github.com/eic/EICrecon/blob/ac3821d47eb4cbd6f39bddaac19710a7840104e0/src/algorithms/tracking/OrthogonalTrackSeedingConfig.h#L79
an explicit quantity is defined in the Acts unit domain. This is the same file you want to add other quantities to in this PR, so it should, for consistency sake, be in the same Acts unit domain, which is manifested by using the Acts::UnitConstants.

In the following lines:
https://github.com/eic/EICrecon/blob/ac3821d47eb4cbd6f39bddaac19710a7840104e0/src/algorithms/tracking/TrackSeeding.cc#L241-L248
this is where the quantities determined in the Acts unit domain are written into EDM4hep unit domain objects. There should be a unit conversion here. In many cases, we have an explicit unit division, so instead of
https://github.com/eic/EICrecon/blob/8469226a42ee9a39170e5ec102913b0020455282/src/algorithms/tracking/TrackSeeding.cc#L242-L247
here we would need:
```
 cov(0,0) = m_cfg.m_loc_a_Error / Acts::UnitConstants::mm; // loc0 
 cov(1,1) = m_cfg.m_loc_b_Error / Acts::UnitConstants::mm; // loc1 
 cov(2,2) = m_cfg.m_phi_Error / Acts::UnitConstants::rad; // phi 
 cov(3,3) = m_cfg.m_theta_Error / Acts::UnitConstants::rad; // theta 
 cov(4,4) = m_cfg.m_qOverP_Error * Acts::UnitConstants::GeV; // qOverP 
 cov(5,5) = m_cfg.m_time_Error / Acts::UnitConstants::ns; // time
```
following the EDM4eic units in https://github.com/eic/EDM4eic/blob/0bd6edd97d0b28b30605ab90dcf338a3244d16dd/edm4eic.yaml#L409-L413.

Reply:
But I will agree that we do need more explicit treatment of EDM4eic units, see https://github.com/eic/EDM4eic/issues/33, https://github.com/eic/EICrecon/issues/925, https://github.com/AIDASoft/podio/pull/445.

Reply:
Thanks for the detailed explanation. I will add the units to both the configuration file and the trackseeding code.